ui/js/controllers/logviewer.js

```diff
@@ -269,10 +274,16 @@ logViewerApp.controller('LogviewerCtrl', [
                             if ($scope.step_data.all_errors.length == 0) {
                                 angular.element(document).ready(function () {
                                     if (isNaN($scope.selectedBegin)) {
-                                        $scope.displayLog($scope.step_data.steps[0], 'initialLoad');
-                                    } else {
-                                        moveScrollToLineNumber($scope.selectedBegin, $event);
+                                        for (var i = 0; i < $scope.step_data.steps.length; i++) {
+                                            var step = $scope.step_data.steps[i];
+                                            if (step.result !== "success") {
+                                                $scope.selectedBegin = step.finished_linenumber;
+                                                $scope.selectedEnd = step.finished_linenumber;
+                                                break;
+                                            }
+                                        }
                                     }
+                                    moveScrollToLineNumber($scope.selectedBegin, $event);
                                 });
                             } else {
                                 $timeout(function() {
```

Comment:
so, basically, empty everything inside the ``$timeout`` block, and use those two lines.  :)